mx.truffleruby/mx_truffleruby.py

```diff
@@ -68,9 +68,14 @@ def ruby_testdownstream(args):
     mx.command_function('build')([])
     mx.run(['ruby', 'tool/jt.rb', 'test', 'fast'])
```

Comment:
I have no real problem with that, but note that if you do so you'll need to ensure some version of MRI is available on the CI machine. Your `ruby_testdownstream_hello` only requires having a JVM and mx available.